docs/src/tutorials/dgm.md

```diff
@@ -0,0 +1,74 @@
+## 2D Poisson's equation
+
+Let's solve the following 2-dimensional Poisson's equation:
+
+```math
+\begin{align*}
+∂^2_x u(x, y) + ∂^2_y u(x, y) = -sin (\pi x) sin (\pi y) \quad & \textsf{for all } 0 < x, y < 1 \, , \\
+u(0, y) = u(1, y) = 0 \quad & \textsf{for all } 0 < y < 1 \, , \\
+u(x, 0) = u(x, 1) = 0 \quad & \textsf{for all } 0 < x < 1 \, , \\
+\end{align*}
+```
+
+We obtain the solution of this equation with the given boundary conditions using Deep Galerkin Method:
+
+```@example dgm_poisson
+using NeuralPDE
+using ModelingToolkit, Optimization, OptimizationOptimisers
+import Lux: tanh, identity
+import ModelingToolkit: Interval, infimum, supremum
+
+@parameters x y
+@variables u(..)
```

Comment:
This writing is for a Poisson example, not a DGM tutorial. Which is it supposed to be? As is it's misclassified in the pages.